src/OgGroupBoolean.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\og;
+
+use Drupal\Core\TypedData\Plugin\DataType\BooleanData;
+
+/**
+ * Overridden boolean data type to hardcode TRUE.
+ */
+class OgGroupBoolean extends BooleanData {
```

Comment:
If this is always `TRUE`, maybe a good name would be `BooleanTrue`? Or `AlwaysTrue`?